### PR TITLE
-Stubbed Observable class

### PR DESCRIPTION
**Pull request by Lrac**
-Moved Observable higher in project structure
-Changed State into two separate state classes and stubbed them
-removed several classes out of namespace

### CODE REVIEW

File: SubZero/src/Observable.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+#ifndef OBSERVABLE_H_
+#define OBSERVABLE_H_
+
+#include <vector>
+#include "Observer.h"
+
+
+/**
+ * Observable is a abstract class that is made known to a observer. It has a state
```
Comment (PyroIgnus) [inline SubZero/src/Observable.h]: Observable no longer has a State

**Comment by PyroIgnus** [comment]
Your commit doesn't actually have the separate State classes added.  It only removes them it looks like.

**Comment by ahsueh1996** [comment]
Ok, so this has not been merged yet right? I don't need to re-fork yet?

**Comment by PyroIgnus** [comment]
Looks good!  If anyone else doesn't have any thing else to bring up then I'll merge this.

**Pull request merged**